src/hapi/pipelines/configs/conflict_event.yaml

```diff
@@ -0,0 +1,829 @@
+    - "conflict_event_gtm_civilian_targeting"
+    - "conflict_event_gtm_demonstration"
+    - "conflict_event_hnd_political_violence"
+    - "conflict_event_hnd_civilian_targeting"
+    - "conflict_event_hnd_demonstration"
+    - "conflict_event_hti_political_violence"
+    - "conflict_event_hti_civilian_targeting"
+    - "conflict_event_hti_demonstration"
+    - "conflict_event_mli_political_violence"
+    - "conflict_event_mli_civilian_targeting"
+    - "conflict_event_mli_demonstration"
+    - "conflict_event_mmr_political_violence"
+    - "conflict_event_mmr_civilian_targeting"
+    - "conflict_event_mmr_demonstration"
+    - "conflict_event_moz_political_violence"
+    - "conflict_event_moz_civilian_targeting"
+    - "conflict_event_moz_demonstration"
+    - "conflict_event_ner_political_violence"
+    - "conflict_event_ner_civilian_targeting"
+    - "conflict_event_ner_demonstration"
+    - "conflict_event_nga_political_violence"
+    - "conflict_event_nga_civilian_targeting"
+    - "conflict_event_nga_demonstration"
+    - "conflict_event_pse_political_violence"
+    - "conflict_event_pse_civilian_targeting"
+    - "conflict_event_pse_demonstration"
+    - "conflict_event_sdn_political_violence"
+    - "conflict_event_sdn_civilian_targeting"
+    - "conflict_event_sdn_demonstration"
+    - "conflict_event_slv_political_violence"
+    - "conflict_event_slv_civilian_targeting"
+    - "conflict_event_slv_demonstration"
+    - "conflict_event_som_political_violence"
+    - "conflict_event_som_civilian_targeting"
+    - "conflict_event_som_demonstration"
+    - "conflict_event_ssd_political_violence"
+    - "conflict_event_ssd_civilian_targeting"
+    - "conflict_event_ssd_demonstration"
+    - "conflict_event_syr_political_violence"
+    - "conflict_event_syr_civilian_targeting"
+    - "conflict_event_syr_demonstration"
+    - "conflict_event_tcd_political_violence"
+    - "conflict_event_tcd_civilian_targeting"
+    - "conflict_event_tcd_demonstration"
+    - "conflict_event_ukr_political_violence"
+    - "conflict_event_ukr_civilian_targeting"
+    - "conflict_event_ukr_demonstration"
+    - "conflict_event_ven_political_violence"
+    - "conflict_event_ven_civilian_targeting"
+    - "conflict_event_ven_demonstration"
+    - "conflict_event_yem_political_violence"
+    - "conflict_event_yem_civilian_targeting"
+    - "conflict_event_yem_demonstration"
+  format: "xlsx"
+  sheet: "Data"
+  admin:
+    - ~
+    - "Admin2 Pcode"
+  admin_exact: True
+  input:
+    - "Month"
+    - "Year"
+    - "Events"
+  list:
+    - "Month"
+    - "Year"
+    - "Events"
+  output:
+    - "Month"
+    - "Year"
+    - "Events"
+  output_hxl:
+    - "#date+month"
+    - "#date+year"
+    - "#event+num"
+
+conflict_event_admintwo:
+  conflict_event_afg_political_violence:
+    dataset: "afghanistan-acled-conflict-data"
+    resource: "afghanistan_hrp_political_violence_events_and_fatalities_by_month-year"
+    input:
+      - "Fatalities"
```

Comment:
Only two of the three datasets have a fatalities column, demonstrations do not!

Reply:
We could consider adding the facility to remove defaults eg. in this case fatalities would go in the defaults and for demonstrations, there would be something to say that there isn't a fatalities column. Alternatively, allow defaults to be specified as optional somehow so that if they're not present they are ignored.